packages/raml-generator/src/main.js

```diff
@@ -0,0 +1,45 @@
+const fs = require('fs');
+const path = require('path');
+const shell = require('shelljs');
+const chalk = require('chalk');
+
+function generateRaml(options) {
+  const resolvedSourcePath = path.resolve(options.apiSpecSourcePath);
+
+  // Check if the spec file exists.
+  fs.access(resolvedSourcePath, fs.constants.F_OK, err => {
+    if (err) {
+      console.error(
+        `${resolvedSourcePath} does not exist`,
+        chalk.red.bold('ERROR')
+      );
+      process.exit(1);
+    }
+
+    generateRamlIfSpecPath({
+      ...options,
+      apiSpecSourcePath: resolvedSourcePath,
+    });
+  });
+}
+
+function generateRamlIfSpecPath(options) {
+  const resolvedDestinationPath = path.resolve(options.apiSpecDestinationPath);
+
+  console.log(
+    `RAML docs will be generated in this directory: ${chalk.blue(
+      resolvedDestinationPath
+    )}`
+  );
+
+  const jarFile = path.resolve(
+    __dirname,
+    '../jar/cli-application-1.0.0-20200221085820-all.jar'
+  );
+
+  shell.exec(
+    `java -jar ${jarFile} generate ${options.apiSpecSourcePath} -o ${resolvedDestinationPath}/${options.apiSpecName} -t ramldoc`
```

Comment:
The script requires users to have `java`. This **needs to be mentioned** in the README! Also which java version etc.